src/components/ProjectSummary.js

```diff
@@ -25,87 +26,91 @@ const ProjectSummary = props => {
             </Segment>
             <Grid columns={2} stackable>
               <Grid.Row>
-                <Grid.Column width={8}>
-                  <Card fluid raised className='project-info-card'>
-                    <Card.Content>
-                      <Card.Header>Connected on</Card.Header>
-                      <Card.Description className='project-info-card-description'>
-                        {project.sourceRepositories.map(repo => {
-                          let array = repo.url.split('/')
-                          let repoName = array[array.length - 1] || array[array.length - 2]
-                          return (
-                            <Fragment key={repo.id}>
+                {(project.pivotaltracker_url || project.sourceRepositories.length > 0 || project.slack_channel_name) ? (
```

Comment:
it is starting to look like a new component is needed here to make it more readable. Nevertheless I am ok merging it like this and refactor it later